node/index.js

```diff
@@ -445,10 +448,27 @@ function TChannelConnection(channel, socket, direction, remoteAddr) {
 
     socket.once('close', clearTimer);
 
-    self.socket
+    var stream = self.socket;
+
+    if (dumpEnabled) {
+        stream = stream.pipe(Spy(process.stdout, {
```

Comment:
this is going to give us nkb of tcp packets. instead of printing nice tchannel frame boundaries.

Is there a way of hooking this up so we hex can show us the frame boundaries ?